modules/jupyter-widget/src/index.js

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Uber Technologies Inc.
+// Distributed under the terms of the Modified BSD License.
+
+export * from './version';
```

Comment:
We usually use explicit exports rather than wildcards. 

Rationale: 
- The top-level index file is a great place to articulate exactly what API surface we expose.
- Also, wildcards can cause "leakage" of internal/experimental syntax.